doc/manuals/admin/database_model.md

```diff
@@ -309,8 +308,8 @@ Fields:
 -   **payload**: optional field to store the payload for notification customization functionality in NGSIv2.
 -   **lastFailure**: the time when last notification failure occurred.
     Not present if the subscription has never failed.
--   **timesFailed**: number of consecutive notification errors for the subscription.
-    Not present if the subscription has never failed.
+-   **lastSuccess**: the time when last successful notification occurred.
+    Not present if the subscription has never provoked a notification.
```

Comment:
Maybe is more precise

"Not present if the subscription has never provoked a sucessful notification"